draft-packages/tabs/KaizenDraft/Tabs/styles.scss

```diff
@@ -99,17 +106,67 @@
   }
 }
 
-.disabledTab {
+.disabledTabRow {
   composes: tab;
+  composes: row;
   cursor: inherit;
   opacity: 0.3;
+  font-weight: normal;
+
+  &.activeTabRow:hover {
+    background-color: transparent;
+  }
 
-  &:hover:not(.activeTab) {
+  &:hover:not(.activeTabRow) {
     color: $kz-color-wisteria-800;
     background-color: transparent;
+    font-weight: normal;
+  }
+}
+
+.row + .row {
+  @include ca-margin($start: $ca-grid * 0.5);
+}
+
+.column {
+  composes: tab;
+  min-height: $kz-spacing-xl;
+  display: flex;
+  align-items: center;
+  justify-content: left;
+  padding-left: $kz-spacing-sm;
+  border-radius: $kz-border-solid-border-radius;
+
+  :global(.js-focus-visible) &:global(.focus-visible):after {
+    border-radius: $kz-border-borderless-border-radius;
+    z-index: 1; // pop the border out so sibling tabs don't overlap
+  }
+
+  &.rtl {
+    justify-content: right;
```

Comment:
Actually, it's better to just use `justify-content: start` and then there's no need to explicitly support RTL - it comes for free!
![image](https://user-images.githubusercontent.com/702885/85973620-b2cc4600-ba16-11ea-8903-95a7084e8b53.png)

Reply:
`justify-content: flex-start`

Reply:
TIL - thanks!

Reply:
Spoke to @sebpearce - `justify-content: flex-start` is better as `justify-content: start` is really new (I thought it was the other way around) and might not work in IE11.